java/src/main/avro/data_row.avsc

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
where is this used?

Reply:
it use to populate POJO before sending to FG

Reply:
or POJO is generated using this schema